### DISCUSSION TITLE
Sortable List Fallback Default Value

### DISCUSSION THREAD

**Discussion by digitaldirk** [microsoft/fluentui-blazor#1601] [General]
Hi,

I wanted to discuss the new Sortable List component fallback property. The fallback property is defaulted to false in this library: https://github.com/microsoft/fluentui-blazor/blob/bb9b53df10b84f97a0653b4b0d7e92cb7853001c/src/Core/Components/SortableList/FluentSortableList.razor.cs#L88
While it is defaulted to true in BlazorSortable: https://github.com/the-urlist/BlazorSortable/blob/5b9ee8db115a296f09638ddc8b4c95e2ad6afe57/Shared/SortableList.razor#L54

I have no issue with this, I was just curious if this was intended and if so what was the reasoning?

Thank you

**Comment by vnbaaij** [comment]
Yes, that was intended. The Urlist team did not like the way the default HTML5 drag and drop worked/was styled andd therefore went with the Sortable provided look and feel. I did not have such strong feelings about it so went with the HTML5 one.

**Reply by digitaldirk** [reply]
Thank you
